store/beestore/beestore_test.go

```diff
@@ -16,13 +18,21 @@ import (
 	"github.com/ethersphere/bee/v2/pkg/swarm"
 	"github.com/gorilla/websocket"
 	"github.com/uncloud-registry/swarm-driver/store/beestore"
-	"github.com/uncloud-registry/swarm-driver/store/teststore"
 )
 
 func TestStoreCorrectness(t *testing.T) {
-	tstore := teststore.NewSwarmInMemoryStore()
-	srvURLStr := newTestServer(t, tstore)
 
+	batchID, err := getNewTestBatchID()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	bstore, err := beestore.NewBeeStore("localhost", 1633, false, batchID, false, true)
```

Comment:
This is a unit test. So it doesnt make sense to run against a running bee instance. If you really want to do this, you should look at [testcontainers](https://testcontainers.com/). With this you need to create a container in the test and start it, then run the test. You cannot expect others to have bee node running when the run the test.

There are plenty of examples on their webpage to do this. You can run the ethswarm/bee image in dev mode and get the test working.